pytorch/pytorch-model-zoo/src/test/resources/mlrepo/model/cv/gan.ai.djl.pytorch.biggan-deep/metadata.json

```diff
@@ -0,0 +1,81 @@
+{
+  "metadataVersion": "0.1",
+  "resourceType": "model",
+  "application": "cv/gan",
+  "groupId": "ai.djl.pytorch",
+  "artifactId": "biggan-deep",
+  "name": "BigGAN-Deep",
+  "description": "BigGAN image generation",
+  "website": "http://www.djl.ai/pytorch/model-zoo",
+  "licenses": {
+    "license": {
+      "name": "The Apache License, Version 2.0",
+      "url": "https://www.apache.org/licenses/LICENSE-2.0"
+    }
+  },
+  "artifacts": [
+    {
+      "version": "0.0.1",
```

Comment:
Shouldn't we have the version as 1.0 ? To match the convention with other datasets ?

Reply:
Use use 1.0 for dataset, since they are pretty mature, and we don't expect much version change down the road.
We use "0.0.x" for models